sites/mosul/configs/openmrs/initializer_config/drugs/drugs.csv

```diff
@@ -1,7 +1,758 @@
+"c83be09f-68b7-4ec1-8616-072bd56d5e57","AS 25mg / AQ eq. 67.5mg base, blister of 3 tab, 4.5-8 kg","9ad20676-eded-4624-ad45-6b411ca939c5","d969ee2e-4052-417c-9954-e2ded7ed0e00","25mg / eq. 67.5mg base, 4.5-8 kg, 3 tablets"
+"7b6cd19f-f2de-4c6a-80bf-ce2a903e9629","AS 50 mg / AQ eq. 135mg base, blister of 3 tab, 9-17 kg","9ad20676-eded-4624-ad45-6b411ca939c5","d969ee2e-4052-417c-9954-e2ded7ed0e00","50 mg / eq. 135mg base, 9-17 kg, 3 tablets"
+"6c97458c-e5cb-4cb7-b550-ad09643c7448","AS 100mg / AQ eq. 270mg base, blister of 3 tab, 18-35 kg","9ad20676-eded-4624-ad45-6b411ca939c5","d969ee2e-4052-417c-9954-e2ded7ed0e00","100mg / eq. 270mg base, 18-35 kg, 3 tablets"
+"a02d71ef-852e-45e8-b329-d7a744c137cf","AS 100mg / AQ eq. 270mg base, blister of 6 tab., >36 kg","9ad20676-eded-4624-ad45-6b411ca939c5","d969ee2e-4052-417c-9954-e2ded7ed0e00","100mg / eq. 270mg base, >36 kg, 6 tablets"
+"877161a8-9c90-4944-a85d-cc5ae9a53ec2","ASCORBIC acid (vitamin C), 50mg, tab.","cf39d27f-6906-403b-843f-f87f5d99a0d1","c1f4a088-32a8-4b34-8162-944bb6ed026f","50mg"
+"8918f542-aa09-47cc-ace4-6147a9bc44dd","ASCORBIC acid (vitamin C), 250mg, chewable tab.","cf39d27f-6906-403b-843f-f87f5d99a0d1","f6c26c40-610e-4ae6-aa35-54a0facac81d","250mg"
+"87dcc7cd-7adf-4ad8-88c9-19b4959d9175","ASCORBIC acid (vitamin C), 500mg, tab.","cf39d27f-6906-403b-843f-f87f5d99a0d1","c1f4a088-32a8-4b34-8162-944bb6ed026f","500mg"
+"0428ac96-4bc6-4f39-b06a-8d58de38c6d1","AS 20mg/ PYRONARIDINE 60mg, granules, sachet","2d7640d8-9fdc-4c2c-abf0-cb6f2f1a50e7","2aba1a18-5eeb-4e2e-9ff3-879580b74286","20mg / 60mg"
+"e67a7447-42ff-46e2-976e-45893f5c5840","AS 60mg/ PYRONARIDINE 180mg, tab.","2d7640d8-9fdc-4c2c-abf0-cb6f2f1a50e7","c1f4a088-32a8-4b34-8162-944bb6ed026f","60mg / 180mg"
+"881028f3-3ecf-4792-8330-c26db80c2e57","ATOVAQUONE 62.5mg/ PROGUANIL HCl 25mg, tab., 11-40kg","3fb4c218-4034-4a9b-bc49-fbaac02e10df","c1f4a088-32a8-4b34-8162-944bb6ed026f","62.5mg/ 25mg, 11-40kg"
+"55be2f89-d4b5-4f62-91d0-5296898b59d2","ATOVAQUONE 250mg / PROGUANIL HCl 100mg, tab., >40 kg","3fb4c218-4034-4a9b-bc49-fbaac02e10df","c1f4a088-32a8-4b34-8162-944bb6ed026f","250mg / 100mg, >40 kg"
+"3e6e85f5-5f6d-45bf-901f-46eec49c967d","ATORVASTATIN calcium, eq. 10mg base, tab.","8ce5a550-5f87-45ae-9bf5-a44ad31cb3fe","c1f4a088-32a8-4b34-8162-944bb6ed026f","eq. 10mg base"
+"808dca1e-2316-413c-9521-7af94103890b","ATORVASTATIN calcium, eq. 20mg base, tab.","8ce5a550-5f87-45ae-9bf5-a44ad31cb3fe","c1f4a088-32a8-4b34-8162-944bb6ed026f","eq. 20mg base"
+"fd251dd6-e51a-4db3-81b4-2e9c8c6490e6","ATORVASTATIN calcium, eq. 40mg base, tab.","8ce5a550-5f87-45ae-9bf5-a44ad31cb3fe","c1f4a088-32a8-4b34-8162-944bb6ed026f","eq. 40mg base"
+"4341f27d-a153-4a53-b964-1cef2c91ad3b","ATV 300mg / r 100mg, tab.","9b180d25-7125-4f6a-ae48-135d10d079f2","c1f4a088-32a8-4b34-8162-944bb6ed026f","300mg / 100mg"

```

Comment:
This is erroring but it's a false positive as `9b180d25-7125-4f6a-ae48-135d10d079f2 exists in OCL [here](https://app.openconceptlab.org/#/orgs/MSF/sources/MSF/?q=9b180d25-7125-4f6a-ae48-135d10d079f2&isTable=true&isList=false&page=1)

<img width="1460" alt="Screenshot 2025-05-29 at 15 42 23" src="https://github.com/user-attachments/assets/49005124-4daa-4584-bc5c-72567f7e21a8" />